rollup.config.js

```diff
@@ -95,6 +95,7 @@ export default {
           module: true,
         }),
     ],
+    preserveEntrySignatures: false,
```

Comment:
what does this do?

Reply:
Produce a warning if it's absent in rollup 2.0